index.html

```diff
@@ -597,6 +597,10 @@ <h3>Free Collaborative learning</h3>
 			            <ul>
                             <li>Learn by doing with <a href="https://www.spingular.com" target="_blank" rel="noopener">Spingular</a>. Join a live collaborative project made with Jhipster!: Spingular Chat w/Websockets</li>
                         </ul>
+                        <h3>Community Tutorials/Blogs</h3>
```

Comment:
@atomfrede lets merge the section `Online trainings (MOOCs)`, `Free Collaborative learning` and `Community Tutorials/Blogs` together into `Community Tutorials` as they all fall under same kind of stuff and currently having different category is bit confusing. So in the end we will have just
- Books
- Community Tutorials

Reply:
:+1: I thought the same already.

Reply:
done

Reply:
Thanks!